controllers/lookup/k8s_client.go

```diff
@@ -0,0 +1,160 @@
+package lookup
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/datainfrahq/druid-operator/apis/druid/v1alpha1"
+	internalhttp "github.com/datainfrahq/druid-operator/pkg/http"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type K8sClient struct {
+	client client.Client
+}
+
+func NewK8sClient(client client.Client) K8sClient {
+	return K8sClient{
+		client: client,
+	}
+}
+
+func (c *K8sClient) FindLookups(ctx context.Context, reports map[types.NamespacedName]Report) (map[ClusterKey]map[LookupKey]Spec, error) {
+	specs := &v1alpha1.DruidLookupList{}
```

Comment:
"specs" is not a clear variable name. Call it druidLookups or something more understandable.
Also down in the for loop, change "spec" to "lookup" so getting to its properties will look beter (e.g. lookup.Spec.DruidClusterName)

Reply:
Addressed in ede7fdf.